hig.js

```diff
@@ -88,19 +88,19 @@ function speak(phrases) {
 		return false;  	
 	}
 	var p = phrases.shift();
-	if(os == 'mac'){  //mac male voice to talk as hockey player
+	if(os == 'darwin'){  //mac male voice to talk as hockey player
 		var v = 'Alex';
-			if (p.name == ANNOUNCER ) { //windows female voice to talk as announcer 
+			if (p.name == ANNOUNCER ) { //mac female voice to talk as announcer 
 				v = 'Samantha';
 		} 
 	}
-	// else if (os=='win32'){ //windows male voice to talk as hockey player
-		// var v = 'David';
-			// if (p.name == ANNOUNCER ) { //linux female voice to talk as announcer 
-				// v = 'Zira';
-		// }
-	// }
-	// else if (os == 'Darwin'){ // Linux Male voice to talk as hockey player
+	else if (os=='win32'){ //windows male voice to talk as hockey player
+		 var v = 'David';
+			 if (p.name == ANNOUNCER ) { //windows female voice to talk as announcer 
+				 v = 'Zira';
+		}
+	}
+	// else if (os == 'linux'){ // Linux Male voice to talk as hockey player
```

Comment:
These comments should be removed before we merge

Reply:
Removed